collect_app/src/main/java/org/odk/collect/android/utilities/MultiFormDownloader.java

```diff
@@ -44,42 +44,29 @@
 import java.io.OutputStream;
 import java.net.URISyntaxException;
 import java.nio.charset.Charset;
-import java.util.ArrayList;
-import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
-import javax.inject.Inject;
-
 import timber.log.Timber;
 
 import static org.odk.collect.android.utilities.FileUtils.LAST_SAVED_FILENAME;
 import static org.odk.collect.android.utilities.FileUtils.STUB_XML;
 import static org.odk.collect.android.utilities.FileUtils.write;
 
-public class MultiFormDownloader implements FormDownloader {
+@Deprecated
+public class MultiFormDownloader {
```

Comment:
I had a good go at chipping away at this but there is far more to be done. I think we need to get to a point where we're not calling this anywhere but `ServerFormDownloader` to really start tearing it apart. There is a lot of logic around task cancellation in here that needs to get the hell out.